src/js/components/Card.js

```diff
@@ -85,21 +113,59 @@ export default class Card extends Component {
       CLASS_ROOT,
       {
         [`${CLASS_ROOT}--direction-${direction}`]: direction,
-        [`${CLASS_ROOT}--selectable`]: (onClick || video)
+        [`${CLASS_ROOT}--selectable`]: (onClick || video),
+        [`${CLASS_ROOT}--${textSize}`]: textSize
       },
       className
     );
 
+    let labelTag;
+    let headingTag;
+    let paragraphSize;
+    let descriptionSize;
+    if (textSize === 'xlarge') {
+      labelTag = 'h3';
+      headingTag = 'h1';
+      paragraphSize = 'xlarge';
+      descriptionSize = 'large';
+    } else if (textSize === 'large') {
+      labelTag = 'h4';
+      headingTag = 'h1';
+      paragraphSize = 'xlarge';
+      descriptionSize = 'large';
+    } else if (textSize === 'medium') {
+      labelTag = 'h4';
+      headingTag = 'h2';
+      paragraphSize = 'large';
+      descriptionSize = 'medium';
+    } else {
+      labelTag = 'h5';
+      headingTag = 'h3';
+      paragraphSize = 'medium';
+      descriptionSize = 'small';
+    }
+
     let onCardClick = onClick;
     if (!onCardClick && video) {
       onCardClick = this._onClick;
     }
 
     const contentContainer = (
       <Box className={`${CLASS_ROOT}__content`} pad="medium">
-        <Heading tag="h5" uppercase={true} margin="none">{label}</Heading>
-        <Heading tag="h2" strong={true}>{heading}</Heading>
-        <Paragraph margin="none">{description}</Paragraph>
+        {label &&
+          <Heading className={`${CLASS_ROOT}__label`}
```

Comment:
I'd rather add sizing to Label, rather than more CSS for Card. I'd like Card to just be a composition of existing Grommet components wherever possible.